.github/workflows/ci.yml

```diff
@@ -81,21 +81,21 @@ jobs:
           taplo fmt --check
       - name: Build Project with try-runtime
         run: cargo build --features try-runtime
+      - name: Runtime build
+        run: cargo build -p node-polkadex-runtime
       - name: Build Project with runtime benchmarks
         run: cargo build --features runtime-benchmarks
       - name: Test weights generation
         run: ./target/debug/polkadex-node benchmark pallet --pallet "*" --extrinsic "*" --steps 2 --repeat 1
       - name: Normal Build
-        run: RUSTFLAGS="-D warnings" cargo build
-      - name: Runtime build
-        run: cargo build -p node-polkadex-runtime
+        run: RUSTFLAGS="-D warnings" cargo build --release
```

Comment:
Why do we need release in CI? it just extends the CI execution time

Reply:
I know that not always release build compiles when develop build is. I remember that we had before problems related to that and IMO it's better to check build close as much as possible to production environment to exclude such regressions and unsustainable release version.